packages/x-charts/src/models/seriesType/line.ts

```diff
@@ -82,6 +82,17 @@ export interface LineSeriesType
    * @default 'none'
    */
   stackOffset?: StackOffsetType;
+  /**
+   * The value of the line at the base of the series area.
+   *
+   * If `baseline` is a number, it will be used as the value.
+   *
+   * - `'min'` the minimum value of the series will be used.
+   * - `'max'` the maximum value of the series will be used.
```

Comment:
IMO it's more interesting to describe the impact than the internal value modification. And the value usecase seems to be very niche compared to the min/max ones


```suggestion
   * - `'min'` the area will fill the space **under** the line.
   * - `'max'` the area will fill the space **above** the line.
   * - `number` the area will fill the space between this value and the line
```